core/src/transaction_meta.rs

```diff
@@ -97,12 +112,13 @@ impl TransactionMeta {
         }
     }
 
-    pub fn destruct(&self) -> (u64, u64, bool, Vec<u8>, usize) {
```

Comment:
This destruct is different from others. `destruct(&self)` vs `destruct(self)`, just a question.

Reply:
Confused +1 ... cc @yangby-cryptape ?

Reply:
Require a reference as input:

https://github.com/nervosnetwork/ckb/blob/858d623fc2c0472fd51b3d1a2181aef70859ee14/protos/src/convert/mod.rs#L53-L64